client/package.json

```diff
@@ -107,6 +107,7 @@
     "json-loader": "^0.5.4",
     "karma": "^1.2.0",
     "karma-chrome-launcher": "^2.0.0 ",
+    "karma-firefox-launcher": "^1.0.0",
```

Comment:
Kann das da weg, wenn jetzt der Chrome verwendet wird? Oder brauchst du den Firefox in dem Fall, dass process.env.TRAVIS === undefined? Was ist process.env.TRAVIS? Ich weiß irgendwie die Prozess-Umgebung vom Travis. Sind das so Pfad- und Umgebungsvariablen? Wie kann es manchmal sein, dass diese Umgebung manchmal definiert & manchmal nicht definiert ist?